Fixed different transforms between xacro and driver

atyshka opened the pull request:
This fixes #469 and #497 

The d435 URDF had an error where there was a non-zero transform for camera_link but the driver output a zero origin for this link. This caused the transforms to shift between starting a robot description and starting the driver. This commit brings the transforms in line with the driver while keeping the visual and collision geometry the same.

atyshka committed:
Fixed different transforms between xacro and driver

atyshka committed:
Merge remote-tracking branch 'origin/development' into development

doronhi committed:
add initial_reset option to rs_multiple_devices.launch

doronhi committed:
Merge branch 'development' of https://github.com/atyshka/realsense into atyshka-development

doronhi committed:
add bottom_screw joint to _d435.urdf.xacro

doronhi reviewed (commented):
The problem needs to be solved and I agree that camera_link is the basis for all the other optical nodes and therefor it should have a (0,0,0) transformation between camera_link and camera_depth_frame.
I feel there should be another joint between the physical mounting point and camera_link.
Would you check out my suggestion in:
https://github.com/doronhi/realsense/tree/atyshka-development (last commit)

If you agree with the concept, please pay attention to the collision section. I didn't attend to it.

atyshka committed:
Fixed d435 collision position

atyshka commented:
@doronhi I rebased your branch onto mine and fixed the collision coordinates. All that needed to be done was shifting with the same units as you did for the mesh. This seems like a good solution, thanks for the input!

doronhi commented:
Thank you, for your patience and persistence.
